mmcv/cnn/bricks/padding.py

```diff
@@ -2,12 +2,11 @@
 from typing import Dict
 
 import torch.nn as nn
+from mmengine.registry import MODELS
 
-from .registry import PADDING_LAYERS
-
-PADDING_LAYERS.register_module('zero', module=nn.ZeroPad2d)
-PADDING_LAYERS.register_module('reflect', module=nn.ReflectionPad2d)
-PADDING_LAYERS.register_module('replicate', module=nn.ReplicationPad2d)
+MODELS.register_module('zero', module=nn.ZeroPad2d)
```

Comment:
The names of the modules are too simple. Maybe we can add some name mapping in the build function, and use the full name in the registry.